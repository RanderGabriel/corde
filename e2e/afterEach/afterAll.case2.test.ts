import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("should print on console in async function", async () => {
  const command = Utils.buildCommandWithConfigPath("afterAll", "bot_case2.test.ts");
  const results = await cli.exec(command);
  expect(results.statusCode).toEqual(1);
  expect(results.stdout).toMatchSnapshot();
});
