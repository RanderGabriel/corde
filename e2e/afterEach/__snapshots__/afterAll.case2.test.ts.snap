// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should print on console in async function 1`] = `
"
 PASS   e2e\\\\afterAll\\\\__cordeTest__\\\\bot_case2.test.ts   100ms
   âœ”  <empty test name>   100ms

Test Files: 1 passed. 1 total
Tests:      1 passed. 1 total
Time:       100ms
test afterAll
"
`;
